app/calculate/index.js

```diff
@@ -0,0 +1,23 @@
+const rates = require('./rates.json')
+
+/**
+ * Calulates the standard payment rate for each ambition
+ * @param {String} code - The land use cover code (standard code)
+ * @param {Array} parcels - The array of eligible parcels allocated for use
+ * @returns {Object} - A payment rates object
+ */
+function calculatePaymentRates (code, parcels) {
+  const paymentRates = {}
+  const rate = rates[`_${code}`] || {}
+  const totalArea = parcels.reduce((a, b) => a + (b.area || 0), 0)
+
+  for (const key in rate) {
+    paymentRates[key] = (totalArea * rate[key]).toFixed(2)
```

Comment:
Thinking about it, I think we need to include the payment rate in the response so the front end team can display how the calculation was derived and what the rate is for each standard.

what about something like

```
{
  "introductory": {
    "rate": 55,
    "paymentAmount": 1000
  }
  ....
{
```